anycable-go/templates/hpa.yaml

```diff
@@ -0,0 +1,23 @@
+{{- $values := include "anycableGo.values" . | fromYaml }}
+{{- with $values.hpa }}
+{{- if .enabled }}
+---
+apiVersion: autoscaling/v1
```

Comment:
I added the template for setting the actual api version [here](https://github.com/anycable/anycable-helm/blob/master/anycable-go/templates/_helpers.tpl#L21)

Please, use it here instead of hard-coding `autoscaling/v1` here